images/cilium/Dockerfile

```diff
@@ -85,10 +80,9 @@ COPY --from=cilium-envoy /usr/bin/cilium-envoy /usr/bin/cilium-envoy-starter /us
 # When used within the Cilium container, Hubble CLI should target the
 # local unix domain socket instead of Hubble Relay.
 ENV HUBBLE_SERVER=unix:///var/run/cilium/hubble.sock
-COPY --from=hubble /out/${TARGETOS}/${TARGETARCH}/bin/hubble /usr/bin/hubble
-COPY --from=hubble /out/linux/bash_completion /etc/bash_completion.d/hubble
-
 COPY --from=builder /tmp/install/${TARGETOS}/${TARGETARCH} /
```

Comment:
Doesn't this line here not also accidentally copy the `hubble` binary to `/` into the destination image?

Reply:
no, because the hubble binary is in `/tmp/hubble`.

Reply:
Of course! :facepalm: Apologies for the noise